index.html

```diff
@@ -688,14 +688,31 @@
               <li>Let <var>identifiers</var> be the first element in the <var>
                 paymentMethod</var> tuple.
               </li>
-              <li>Let <var>data</var> be the second element in
-              <var>paymentMethod</var> tuple.
+              <li>Let <var>data</var> be the result of <a data-cite=
+              "!ECMA-262-2015#sec-json.parse">JSON-parsing</a> the second
+              element in <var>paymentMethod</var> tuple.
               </li>
-              <li>Determine which <a>payment handlers</a> support any of the
-              <a>payment method identifiers</a> given <var>identifiers</var>.
-              For each resulting payment handler, if payment method specific
-              capabilities supplied by the payment handler match those provided
-              by <var>data</var>, the payment handler matches.
+              <li>Let <var>methods</var> be a list of <a>payment methods</a>
+              that support <var>identifiers</var>.
```

Comment:
Will switch it back. 

@adrianhopebailie, please provide code counter examples. If you can't show actual code examples then we can't have a discussion.